### PR TITLE
Update READMEs

### PR DESCRIPTION
**Pull request by Gittenburg**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,24 +6,24 @@
...
+* **Learn with spaced-repetition** (our algorithm is inspired by Anki & SuperMemo2)
+* **Markdown, LaTeX, images**
+* **Share & collaborate**: Decks are public and can be edited by anyone.
+* **History**: Revisions can be easily reviewed and reverted.
+* **Organize** cards into categories.
+* **Remix**: both decks and cards can be easily copied.
+* **No password**: Login with GitHub, GitLab or Google
 
-## Howto user story
+## Contributing
```
Comment (Otto-AA) [inline README.md:19]: We could add our technology stack here (spring-boot, angular)

**Comment by Gittenburg** [comment]
With GitHub the README is prominently displayed, while CONTRIBUTING.md isn't, it's just linked when creating a pull request or an issue, so we definitely want a contributing section in our README to visibly encourage contributions. I think coding guidelines should be enforced by the CI rather than written out verbally.

**Comment by Otto-AA** [comment]
While the CI should enforce the style guide, I think it would be good to highlight the way to manually check and fix linting errors. For instance, Angular provides the commands `ng lint` and `ng lint --fix` for linting the source code. I'm not sure what the equivalent commands would be for the backend, but I'm confident we could setup automatic linting and fixing for it.

**Comment by Otto-AA** [comment]
Regarding the contributing section, I would suggest to have a minimal section in the readme which links to the CONTRIBUTING file with a more detailed description.

**Pull request merged**